packages/ckb/src/types/queue.ts

```diff
@@ -0,0 +1,16 @@
+import { Hex } from './common';
+import { BaseCkbVirtualTxResult } from './rgbpp';
+
+export interface AppendRgppTxRequest {
+  txid: Hex;
+  ckbVirtualResult: BaseCkbVirtualTxResult;
+}
+
+export interface AppendRgppTxResp {
+  state: string;
+}
+
+export interface CheckRgppTxResp {
+  ckbTxHash: string;
+  state: Hex;
```

Comment:
The `state` is a string that can have one of the following values: 

- completed: The CKB transaction has been sent and confirmed.
- failed: Something went wrong during the process, and it has failed.
- delayed: The transaction has not been confirmed yet and is waiting for confirmation.
- active: The transaction is currently being processed.
- waiting: The transaction is pending and is waiting to be processed.

I will add these to the API documents later.

Reply:
Updated https://github.com/ckb-cell/rgbpp-sdk/pull/28/commits/5c280f279831bab64b6794888f0faf3f926c40e1